tests/openwisp2/sample_radius/apps.py

```diff
@@ -1,7 +1,19 @@
 from openwisp_radius.apps import OpenwispRadiusConfig
+from openwisp_radius.receivers import radius_accounting_success_handler
+from openwisp_radius.signals import radius_accounting_success
 
 
 class SampleOpenwispRadiusConfig(OpenwispRadiusConfig):
     name = 'openwisp2.sample_radius'
     label = 'sample_radius'
     verbose_name = 'Sample Radius'
+
+    def connect_signals(self):
+        from .api.views import AccountingView
+
+        radius_accounting_success.connect(
+            radius_accounting_success_handler,
+            sender=AccountingView,
+            dispatch_uid='radius_accounting_success',
+        )
+        return super().connect_signals()
```

Comment:
To test the signal is calling the reciever I added the test, but in sample_app it was failing because the actual view method changes which calls the signal. So to make it pass I have to define it here as per [this comment](https://github.com/openwisp/openwisp-radius/pull/355#discussion_r754318693).